var/spack/repos/builtin/packages/sendme/package.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Sendme(CargoPackage):
+    """A cli tool to send directories over the network, with NAT hole punching"""
+
+    homepage = "https://www.iroh.computer/sendme"
+    git = "https://github.com/n0-computer/sendme.git"
+
+    maintainers("n0-computer")
+
+    license("Apache-2.0 OR MIT")
+
+    version("main", branch="main")
+    version("0.23.0", tag="v0.23.0", commit="39f6111d8c3a973ea1f54a3c47aad07014de854b")
+
+    def test_sanity(self):
+        """Verify that the 'sendme' binary is available"""
+        sendme = which(self.prefix.bin.sendme)
+        output = sendme("-V", output=str.split, error=str.split)
+        assert "sendme" in output, "Expected 'sendme' in the output"
```

Comment:
```suggestion
    @run_after("install")
    def test_sanity(self):
        """Verify that the 'sendme' binary is available"""
        sendme = which(self.prefix.bin.sendme)
        output = sendme("-V", output=str.split, error=str.split)
        assert "sendme" in output, "Expected 'sendme' in the output"
```

We should either add the `@run_after("install")` decorator, or use Spack's integrated [sanity checks](https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-sanity-checks) like so,

```
sanity_check_is_file = ["bin/sendme"]
```

Reply:
I wasn't aware of the sanity_check_is_file option—thanks for pointing it out! Would it be an issue if I included both that and the @run_after("install") decorator, or would that be redundant?

Reply:
That'd probably be a little redundant. Following from other packages I'd recommend just setting the `sanity_check_is_file` property near the top of the file. Here's another package as an example,

https://github.com/spack/spack/blob/f5aa15034e580a8697da162dfa6ef32bfecf29ee/var/spack/repos/builtin/packages/fzf/package.py#L23